### ISSUE TITLE
It seems that global step (tf.train.get_global_step) has not been increased

### ISSUE THREAD

**Issue by guotong1988** [google-research/bert#915]
Version: tensorflow-gpu 1.14.0
Run:`run_classifier.py`

```
WARNING:tensorflow:It seems that global step (tf.train.get_global_step) has not been increased. Current value (could be stable): 10033 vs previous va     lue: 10033. You could increase the global step by passing tf.train.get_global_step() to Optimizer.apply_gradients or Optimizer.minimize.
 952 W1114 11:00:33.829411 140169788213056 basic_session_run_hooks.py:724] It seems that global step (tf.train.get_global_step) has not been increased. Cu     rrent value (could be stable): 10033 vs previous value: 10033. You could increase the global step by passing tf.train.get_global_step() to Optimizer.     apply_gradients or Optimizer.minimize.
```

**Comment by xyz010** [comment]
check also this [issue](https://github.com/NVIDIA/DeepLearningExamples/issues/80) it has to do with a NaN loss during training. It is not necessarily bad!
